pages/admin/users.tsx

```diff
@@ -1,5 +1,95 @@
 import DashboardLayout from "components/DashboardLayout";
-import { useState } from "react";
+import { useEffect, useState } from "react";
+
+const UI_PAGE_SIZE = 7;
+const BACKEND_PAGE_SIZE = 21;
+
+const getBackendPageNumber = (uiPage: number): number[] => {
+  const pageNum = Math.floor((uiPage * UI_PAGE_SIZE) / BACKEND_PAGE_SIZE);
+  const startIndex = UI_PAGE_SIZE * (uiPage % 3);
+  return [pageNum, startIndex];
+};
+
+interface User {
+  name: string;
+  email: string;
+  image: string;
+  phoneNumber: string;
+}
+
+const UserDashboardItem: React.FunctionComponent<User> = ({
```

Comment:
great job with the dashboard and dashboard components! we're hoping to reuse these components across different parts of the app, so would it be possible to 1) turn these into components and move them to the /components folder and 2) use these components in the users page?